src/Models/NonhydrostaticModels/nonhydrostatic_model.jl

```diff
@@ -119,7 +125,8 @@ function NonhydrostaticModel(; grid,
             particles::ParticlesOrNothing = nothing,
     biogeochemistry::AbstractBGCOrNothing = nothing,
                                velocities = nothing,
-                                pressures = nothing,
+             hydrostatic_pressure_anomaly = nothing,
+                  nonhydrostatic_pressure = CenterField(grid),
```

Comment:
Just to follow the same order of args as in the docstrings:

```suggestion
                  nonhydrostatic_pressure = CenterField(grid),
             hydrostatic_pressure_anomaly = nothing,
```

Reply:
Should we just rename `hydrostatic_pressure_anomaly` to `hydrostatic_pressure` for simplicity?

Reply:
I'm open to that, let's discuss.

The possible confusion of that simplification is that there are other parts of the hydrostatic pressure (eg associated with reference density). The distinction between physical pressure and dynamic / modeled pressure is a persistent source of confusions so...

Even "hydrostatic" is a little confusing because this is, of course, dynamic.

Reply:
So sorry to be more specific: we call it the "anomaly" because the total hydrostatic pressure includes, for example, `rho0 * g * z` (there is also an additional component to the true pressure that we ignore, which is associated with compressibility effects when using a nonhydrostatic equation of state)

Reply:
Yeah, I agree with that. I suggested the change mainly because when we say `hydrostatic_pressure`, people will correctly assume what we're calculating there. I think when users see `hydrostatic_pressure_anomaly`, it won't be obvious to them what it is. But maybe that's for the best...?

I don't feel strongly about it though.

Reply:
You saying that if uses write

```julia
hydrostatic_pressure = CenterField(grid)
```

that it will be obvious that `hydrostatic_pressure` isn't the total hydrostatic pressure but rather just the anomaly associated with buoyancy forces?

Reply:
I don’t really have a strong opinion. But personally with “hydrostatic” I would thing the ρ g z part was included

Reply:
I think we should keep anomaly for now since its consistent with the docs and whatnot. In the future we can consider revising the docs + this kwarg.